coverage/generate-index-html.py

```diff
@@ -27,97 +29,69 @@
-            <th>JS Coverage</th>
-            <th>C++ Coverage</th>
-          </tr>
-        </thead>
-      <tbody>
+    <div class="container">
+      <h1>Node.js Nightly Code Coverage</h1>
+      <h3>
+        Node.js Core&nbsp;&nbsp;<a href="https://github.com/nodejs/node">&rarr;</a>
+      </h3>
+      <main>
+        <div class="page-content">
+          <div class="table">
+            <div class="table-header">
+              <div>Date</div>
+              <div>HEAD</div>
+              <div>JS Coverage</div>
+              <div>C++ Coverage</div>
+            </div>
 ''')
   for line in reversed(index_csv):
     jscov, cxxcov, date, sha = line.split(',')
+    date = datetime.datetime.strptime(date, '%Y-%m-%dT%H:%M:%S%fZ').strftime("%d/%m/%Y %H:%M")
     out.write('''
-          <tr>
-            <td>{0}</td>
-            <td><a href="https://github.com/nodejs/node/commit/{1}">{1}</a></td>
-            <td><a href="coverage-{1}/index.html">{2:05.2f}&nbsp;%</a></td>
-            <td><a href="coverage-{1}/cxxcoverage.html">{3:05.2f}&nbsp;%</a></td>
-          </tr>'''.format(date, sha, float(jscov), float(cxxcov)))
+            <div class="table-row">
+              <div>{0}</div>
+              <div><a href="https://github.com/nodejs/node/commit/{1}">{1}</a></div>
+              <div><a href="coverage-{1}/index.html">{2:05.2f}&nbsp;%</a></div>
+              <div><a href="coverage-{1}/cxxcoverage.html">{3:05.2f}&nbsp;%</a></div>
+            </div>'''.format(date, sha, float(jscov), float(cxxcov)))
   out.write('''
-          </tbody>
-        </table>
+          </div>
+        </div>
       </div>
-   </div>
- </div>
-    <script defer src="https://code.getmdl.io/1.1.3/material.min.js"></script>
+    </div>
     <footer class="no-margin-top">
-
-        <div class="linuxfoundation-footer">
-            <div class="container">
-                <a class="linuxfoundation-logo" href="http://collabprojects.linuxfoundation.org">Linux Foundation Collaborative Projects</a>
-
-                <p>� 2016 Node.js Foundation. All Rights Reserved. Portions of this site originally � 2016 Joyent. </p>
-                <p>Node.js is a trademark of Joyent, Inc. and is used with its permission. Please review the <a href="https://nodejs.org/static/documents/trademark-policy.pdf">Trademark Guidelines of the Node.js Foundation</a>.</p>
-                <p>Linux Foundation is a registered trademark of The Linux Foundation.</p>
-                <p>Linux is a registered <a href="http://www.linuxfoundation.org/programs/legal/trademark" title="Linux Mark Institute">trademark</a> of Linus Torvalds.</p>
-            </div>
+      <div class="linuxfoundation-footer">
+        <div class="container">
+          <a class="linuxfoundation-logo" href="http://collabprojects.linuxfoundation.org">Linux Foundation Collaborative Projects</a>
+          <p>&copy; 2016 Node.js Foundation. All Rights Reserved. Portions of this site originally &copy; 2016 Joyent. </p>
```

Comment:
Seems though that the benchmarking page has the same header.  Looking at the files when I download them, the difference may be an extra character so what I think it happening is that some sort of escape is being lost since we print out the string as opposed to it just being in the file.  

Having said all that I think instead of figure out how to escape that character, the @copy; is much clearer when looking at the source so I'd prefer to go with that.